src/components/ContentNode.vue

```diff
@@ -100,29 +100,35 @@ function renderNode(createElement, references) {
     ))
   ));
 
-  const renderTableChildren = (rows, headerStyle = TableHeaderStyle.none) => {
+  const renderTableCell = (
+    element, attrs, data, cellIndex, rowIndex, extendedData,
+  ) => {
+    const { colspan, rowspan } = extendedData[`${rowIndex}_${cellIndex}`] || {};
+    // if either is `0`, then its spanned over and should not be rendered
+    if (colspan === 0 || rowspan === 0) return null;
```

Comment:
I'm not entirely sure that we _need_ to add "0" data for anything in the JSON since we can calculate everything by using the indices of the actual cell that is doing the spanning, but this works and makes the code a little shorter I suppose.